ui/raidboss/data/05-shb/raid/e8s.js

```diff
@@ -6,7 +6,193 @@
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E8S Biting Frost',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D66', capture: false }),
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'E8S Driving Frost',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D67', capture: false }),
+      alertText: {
+        en: 'Away From Back',
+      },
+    },
+    {
+      id: 'E8S Diamond Frost',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D6C', capture: false }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E8S Diamond Frost Cleanse',
+      regex: Regexes.ability({ source: 'Shiva', id: '4D6C', capture: false }),
+      suppressSeconds: 1,
+      condition: function(data) {
+        return data.CanCleanse();
+      },
+      infoText: {
+        en: 'Cleanse',
+      },
+    },
+    {
+      id: 'E8S Double Slap',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D65' }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBusterSwap(),
+    },
+    {
+      id: 'E8S Axe Kick',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D6D', capture: false }),
+      response: Responses.getOut(),
+    },
+    {
+      id: 'E8S Scythe Kick',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D6E', capture: false }),
+      response: Responses.getUnder(),
+    },
+    {
+      id: 'E8S Light Rampant',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D73', capture: false }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E8S Banish III',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D80', capture: false }),
+      alertText: {
+        en: 'Stacks',
+      },
+    },
+    {
+      id: 'E8S Banish III Fake',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D81', capture: false }),
+      response: Responses.spread(),
+    },
+    {
+      id: 'E8S Wyrm\'s Lament',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D7C', capture: false }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.aoe(),
+      run: function(data) {
+        data.wyrmsLament = data.wyrmsLament || 0;
+        data.wyrmsLament++;
+      },
+    },
+    {
+      id: 'E8S Wyrm\'s Lament Counter',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4D7C', capture: false }),
+      run: function(data) {
+        data.wyrmsLament = data.wyrmsLament || 0;
+        data.wyrmsLament++;
+      },
+    },
+    {
+      id: 'E8S Wyrmclaw',
+      regex: Regexes.gainsEffect({ effect: 'Wyrmclaw' }),
+      condition: Conditions.targetIsYou(),
+      preRun: function(data, matches) {
+        if (data.wyrmsLament == 1) {
+          data.wyrmclawNumber = {
+            '14.00': 1,
```

Comment:
This is not quite good enough.  Can you make this a conditional for <=14, <=22, <=30, else to assign? Sometimes effect timers are slightly off because the ffxiv plugin returns you the time remaining and it may be delayed, so you might be 13.98s.  This is why Nael doom timers do this as well.